tests/e2e/Pages/ChatComponent.ts

```diff
@@ -153,6 +182,17 @@ class ChatComponent extends DashboardPage {
     await expect(newPage).toHaveURL(url);
     await newPage.close();
   }
+
+  async chatPredictionButtonsAreActive() {
```

Comment:
maybe rename to 'showPredictionButtonsAreActive'? It's already in ChatComponent.ts, so no need to repeat chat maybe.

Reply:
Changed to: 'predictionButtonsAreActive'